cmd/torpedo/torpedo.go

```diff
@@ -657,6 +680,7 @@ func fireTorpedo(c *cli.Context) {
 	v := c.String("storage")
 	n := c.String("node-driver")
 	tests := c.String("tests")
+	logLoc := c.String("logLocation")
```

Comment:
The flag name appears to be `log-location`. Why are you getting the value from `logLocation`?

Reply:
Missed it. My bad.